pkg/oomstore/types/types.go

```diff
@@ -1,10 +1,17 @@
 package types
 
+type Category string
+
 const (
-	BatchFeatureCategory  = "batch"
-	StreamFeatureCategory = "stream"
+	BatchCategory  Category = "batch"
```

Comment:
we do `BackendPostgres` for backend, shall we do `CategoryBatch` for category?

Reply:
sound good! updated.